### ISSUE TITLE
account system like on invidious?

### ISSUE THREAD

**Issue by notthefox** [zedeus/nitter#202]
I love this project so much! I prefer it more then normal twitter but is there anyway to make list or something? i want to use nitter as my normal feed.

**Comment by trymeouteh** [comment]
Please add accounts so we can follow people and have a feed of tweets on the website and in a private RSS feed.

**Comment by Joshndroid** [comment]
This is literally the only thing I would like. 
I am trying to embrace rss feeds but that introduces more difficulties.
I literally just want twitter but private, much like invidious is youtube, but private.
+1 for an account/feed setup

**Comment by paulverbeke** [comment]
There is now a NewPipe-style Front-end Android app equivalent for Twitter: Fritter  https://github.com/jonjomckay/fritter
With local subscriptions, feed etc

**Comment by paulverbeke** [comment]
There is now a NewPipe-style Front-end Android app equivalent for Twitter: Fritter  https://github.com/jonjomckay/fritter
With local subscriptions, feed etc

**Comment by paulverbeke** [comment]
There is now a NewPipe-style Front-end Android app equivalent for Twitter: Fritter  https://github.com/jonjomckay/fritter
With local subscriptions, feed etc

**Comment by paulverbeke** [comment]
There is now a NewPipe-style Front-end Android app equivalent for Twitter: Fritter  https://github.com/jonjomckay/fritter
With local subscriptions, feed etc

**Comment by paulverbeke** [comment]
There is now a NewPipe-style Front-end Android app equivalent for Twitter: Fritter  https://github.com/jonjomckay/fritter
With local subscriptions, feed etc

**Comment by Joshndroid** [comment]
While this is awesome and I am going to check it out. I also would like something similar for my desktop where I use twitter mostly. However I will be checking that app out as well.

**Comment by Loki-L1130** [comment]
Implementing Subscriptions like Libreddit in the Browser would be great. Which is saved in the cookies and can be recovered using a URL in another device.

**Comment by Loki-L1130** [comment]
Instead of accounts Method this implementation requires user to take care of their subscriptions and recovered using a URL and also do not require any data to be save on any instances.

**Comment by Loki-L1130** [comment]
Implementing Local Subscriptions like Libreddit in the Browser would be great. Which saves necessary data in the cookies and can be recovered using a URL in another device.

Instead of accounts Method this implementation requires user to take care of their subscriptions and recovered using a URL and also do not require any data to be save on any instances. 

I think most will agree on this and eliminates the need for a User Database.

**Comment by zedeus** [comment]
Cookie-based custom list is the plan, with ways to export/import it.

**Comment by Pentaphon** [comment]
> Cookie-based custom list is the plan, with ways to export/import it.

So basically like Newpipe. That's good enough for me.

**Comment by Relected** [comment]
yeah I would also like to have the ability to have a local subscription feed, especially since fritter stopped being updated on fdroid for over 6 months

there is also an unofficial nitter android app on fdroid which is very good

**Comment by nickelswitte** [comment]
I would also be very grateful to see this feature, as I'd like to use Nitter as a full time alternative to Twitter

**Comment by rifuller** [comment]
I would like this feature too. Implementation could be client side using cookies and saved as a bookmark.

**Comment by helmut72** [comment]
Since no 3rd party apps are working anymore, it would be really cool if Nitter supports own user accounts to create an own timeline.
It would be perfect if access can be restricted to authenticated users and if some kind of external authentication is supported like OIDC or a header based authentication.

**Comment by Relected** [comment]
afaik nitter.poast.org has something similar, although it works similarly to newpipe and it saves follows locally

**Comment by MasteringTheMess** [comment]
I also very much would like to see something similar to accounts/personalized feed

**Comment by timreuscher** [comment]
Now that search is no longer working (perhaps permanently), and with it viewing multiple accounts at once by simply adding commas, this enhancement seems even more important.

**Comment by lban2049** [comment]
Thank you very much for this project. The page is cleaner and the timeline is clearer! I hope there will be a function to manage subscribed accounts. The discussions above meet my requirements, but I don't know if there is a development plan?

**Comment by EngineerGlitch** [comment]
@zedeus I was wondering when the account system like invidious or piped (or local cookie if not instance-side) would be made to follow people and bookmarks tweets

**Comment by dumbu3** [comment]
any news on this??
